src/main/java/com/JessZuchowski/codefellowship/controllers/FollowController.java

```diff
@@ -0,0 +1,45 @@
+package com.JessZuchowski.codefellowship.controllers;
+
+import com.JessZuchowski.codefellowship.database.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.Date;
+
+@Controller
+@RequestMapping("/feed")
+public class FollowController {
+
+    @Autowired
+    PostRepository postRepo;
+
+    @Autowired
+    FollowRepository followRepo;
+
+    @Autowired
+    UserRepository userRepo;
+
+    @GetMapping("/test")
+    public String test() {
+        ApplicationUser user1 = new ApplicationUser("Yew");

```

Comment:
AppUser does not take in Strings. Instead it should be user1.setFirstName("Yew"); Same for user2.